app/ui/browser-modern/views/browser/page/net-error.jsx

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import React, { PropTypes, Component } from 'react';
+
+import Style from '../../../../shared/style';
+
+const CONTAINER_STYLE = Style.registerStyle({
+  width: '700px',
+  margin: '0 auto',
+  display: 'block',
+});
+
+class NetErrorPage extends Component {
+  render() {
+    return (
+      <div className={CONTAINER_STYLE}>
+        <h1>Could not connect</h1>
+        <p>Tofino can't find the server at <strong>{this.props.url}</strong></p>
```

Comment:
This was just an error page thrown together in the previous front end, we don't have any mockups so this was just made so we don't have a blank page.